### PR TITLE
Weather APP Using API

### PR DESCRIPTION
**Pull request by AndresFMoya**
Good day TSEs.

- GetWeather function was created to fetch OpenWeatherMap API data for location entered by the user.
- GetImage function was created to fetch Giphy API Data depending on the weather.
- A select option was added to choose between Celsius and Fahrenheit.
- Bootstrap was used to style the form.

Looking to hear your comments.

Best!

### CODE REVIEW

File: src/framework.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import DashboardController from './controllers/dashboard.controller';
+
+class Framework {
+  /**
+   * Select the route handler and call App#render.
```
Comment (bellom) [inline src/framework.js]: **Kindly remove commented code, you can always use git history to access previous code right :slightly_smiling_face:**

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @AndresFMoya  :clap: 
#### Awesome job so far!! :+1: 
#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1:, just few issues left to work on............ :slightly_smiling_face: 

Below are part of the checklist for this project, kindly work on these few things listed below;

1. Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

2. You are using `drop down` checkout the screenshot from your web page below;
image


You should use a toggle control to determine if the result will be shown in Celsius or Fahrenheit ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by AndresFMoya** [comment]
Hi @bellom. Thanks for your feedback! The comments were deleted and a toggle control was added to switch between units.

Looking to hear your comments.

Best!

**Review by bellom — APPROVED** [review]
## Hi @AndresFMoya  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Goodluck in the next project :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**